AWSAuthSDK/Sources/AWSMobileClient/AWSMobileClient.swift

```diff
@@ -83,7 +83,6 @@ final public class AWSMobileClient: _AWSMobileClient {
         queue.maxConcurrentOperationCount = 1
         return queue
     }()
-    internal let tokenFetchLock = DispatchGroup()
```

Comment:
tokenFetchLock is used in another place as well, in `handleCognitoAuthGetSession`, called from `showSignIn`